pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -192,36 +200,60 @@ func handleErr(reqLogger logr.Logger, status *cv1beta1.CincinnatiStatus, reason
 	reqLogger.Error(e, reason)
 }
 
+// handleCACertStatus logs the message and sets an appropriate Condition on the ConditionRegistryCACertFound status.
+func handleCACertStatus(reqLogger logr.Logger, status *cv1beta1.CincinnatiStatus, reason string, message string) {
+	conditionsv1.SetStatusCondition(&status.Conditions, conditionsv1.Condition{
+		Type:    cv1beta1.ConditionRegistryCACertFound,
+		Status:  corev1.ConditionFalse,
+		Reason:  reason,
+		Message: message,
+	})
+	reqLogger.Info(message)
+}
+
 func (r *ReconcileCincinnati) ensureAdditionalTrustedCA(ctx context.Context, reqLogger logr.Logger, instance *cv1beta1.Cincinnati, resources *kubeResources) error {
 	// Check if the Cluster is aware of a registry requiring an
 	// AdditionalTrustedCA
 	image := &apicfgv1.Image{}
 	err := r.client.Get(ctx, types.NamespacedName{Name: defaults.ImageConfigName, Namespace: ""}, image)
 	if err != nil && errors.IsNotFound(err) {
+		m := fmt.Sprintf("image.config.openshift.io not found for (Name: %v, Namespace: %v)", defaults.ImageConfigName, "")
+		handleCACertStatus(reqLogger, &instance.Status, "EnsureAdditionalTrustedCAFailed", m)
 		return nil
 	} else if err != nil {
 		return err
 	}
 
 	if image.Spec.AdditionalTrustedCA.Name == "" {
+		m := fmt.Sprintf("image.config.openshift.io.Spec.AdditionalTrustedCA.Name not found for image (Name: %v, Namespace: %v)", defaults.ImageConfigName, "")
+		handleCACertStatus(reqLogger, &instance.Status, "EnsureAdditionalTrustedCAFailed", m)
 		return nil
 	}
 
 	// Search for the ConfigMap in openshift-config
 	sourceCM := &corev1.ConfigMap{}
 	err = r.client.Get(ctx, types.NamespacedName{Name: image.Spec.AdditionalTrustedCA.Name, Namespace: openshiftConfigNamespace}, sourceCM)
 	if err != nil && errors.IsNotFound(err) {
-		reqLogger.Info("Found image.config.openshift.io.Spec.AdditionalTrustedCA.Name but did not find expected ConfigMap", "Name", image.Spec.AdditionalTrustedCA.Name, "Namespace", openshiftConfigNamespace)
+		m := fmt.Sprintf("Found image.config.openshift.io.Spec.AdditionalTrustedCA.Name but did not find expected ConfigMap (Name: %v, Namespace: %v)", image.Spec.AdditionalTrustedCA.Name, openshiftConfigNamespace)
+		handleCACertStatus(reqLogger, &instance.Status, "EnsureAdditionalTrustedCAFailed", m)
 		return err
```

Comment:
In this case, the reconcile throws and err and tries again. So you get this instead:
```
status:
    conditions:
    - lastHeartbeatTime: "2020-05-06T20:56:53Z"
      lastTransitionTime: "2020-05-06T20:56:53Z"
      reason: Reconcile started
      status: "False"
      type: ReconcileCompleted
    - lastHeartbeatTime: "2020-05-06T20:56:53Z"
      lastTransitionTime: "2020-05-06T20:56:53Z"
      message: 'Found image.config.openshift.io.Spec.AdditionalTrustedCA.Name but
        did not find expected ConfigMap (Name: trusted-ca, Namespace: openshift-config)'
      reason: EnsureAdditionalTrustedCAFailed
      status: "False"
      type: RegistryCACertFound
```